pennylane/__init__.py

```diff
@@ -29,7 +29,8 @@
 import pennylane.templates
 from pennylane.templates import template
 from pennylane.about import about
-
+import pennylane.vqe
```

Comment:
Is there anything in `vqe` other than `Hamiltonian` and `VQECost`?

Reply:
Not currently. It makes for nice organization though for future growth.

Reply:
but do we need those things to be accessible to the user at `qml.vqe.XYZ` though?

Reply:
I think this is the final unresolved comment, have removed this import statement.